### PR TITLE
Adds useImageSize hook

### PR DESCRIPTION
**Pull request by barakplasma**

### CODE REVIEW

File: hooks.json [hunk 1]
```diff
@@ -671,6 +671,12 @@
     "repositoryUrl": "https://github.com/infodusha/react-hook-size",
     "importStatement": "import { useSize } from \"react-hook-size\";"
   },
+  {
+    "name": "useImageSize",
+    "tags": ["Browser-API", "fetch"],
+    "repositoryUrl": "https://github.com/use-hooks/react-hooks-image-size",
+    "importStatement": "import useImageSize from '@use-hooks\/image-size';\";"
```
Comment (stereobooster) [inline hooks.json]:
```suggestion
    "importStatement": "import useImageSize from '@use-hooks/image-size';"
```

Comment (nikgraf) [reply hooks.json]: 👍

**Comment by barakplasma** [comment]
Feel free to make any further changes.
Thanks for maintaining the project

**Comment by nikgraf** [comment]
thx 👏

**Pull request merged**